src/bun.js/bindings/ScriptExecutionContext.cpp

```diff
@@ -211,11 +211,11 @@ void ScriptExecutionContext::dispatchMessagePortEvents()
 
 void ScriptExecutionContext::checkConsistency() const
 {
-    for (auto* messagePort : m_messagePorts)
```

Comment:
Why was this commented out?

Reply:
an assertion in scriptExecutionContext failed locally.

reproduce on main with `bun-debug test worker`

Reply:
ideally all of these isMainThread checks would be done, but we have most commented out.

seems to work fine but i feel like webkit had a good reason to